### PR TITLE
Updating instructions & packages/dependencies.

### PR DESCRIPTION
**Pull request by JacOng17**
Updating instructions & packages/dependencies.

Instructions & scripts need to be finalised but just to get something up & the security updates I created this PR to address them.

### CODE REVIEW

**Comment by JacOng17** [comment]
It works locally now. Next step is to test the instructions for docker.

File: bin/setup [hunk 1]
```diff
@@ -1,41 +1,106 @@
...
+  end
 end
+
+def die(message)
+  puts
+  log(:red, "FAIL #{message}", $stderr)
+  exit(1)
+end
+
+setup!
```
Comment (mermop) [inline bin/setup]: This is a lot of changes - is it copied from somewhere, or have you written it all new?

Comment (JacOng17) [reply bin/setup]: From Croissant + some of the previous code. :stuck_out_tongue:

**Comment by Br3nda** [comment]
Maybe it's time to put the code into a reusable module to use in multiple places

File: .travis.yml [hunk 2]
```diff
@@ -4,19 +4,50 @@ cache:
...
+    - google-chrome-stable
+
+env:
+  global:
+    - RAILS_ENV=test
+    - OPENFISCA_URL=https://api.rules.nz
+    - GITHUB_URL=https://github.com/ServiceInnovationLab/openfisca-aotearoa
+    - OPENFISCA_GIT_CLONE_URL=https://github.com/ServiceInnovationLab/openfisca-aotearoa
+    - secure: T7+npfKb6EW0G++OP4C91y2WMrMYyOrozeNT1WPCuedd5vKdILSskMz5LBav54F0J6OBbwozH82e0xclMTa23VuOrc9805tEcwtSRdDYb/mIb7F6gEyseqLIk/TZvDfmB5Yrrji0IVX9s5q0OKfVVb0rTrgh87pSRW2YnYAu41k6v2pK/OquixQX0sogblOLfKYprqh4h3Ay6+1IIDmC/lg0XSe+0xRFBZ5FJk0ycStu74xM+qbphIFgzd6q2kChFqRrOporeCgN3WSAFsQd7QneLITN3eucH9AR0OC9Udpuxr8tvAqTfy7Lo3ogj9PTYjD60ObQjcr212d/isSaZd6Zk2wHH35bzsZU3Y5f6vsILiZVJY2rDPkf7A6jMrWSPw8jd/CRizFjedveF9jCy3J2viwNI23y4Ce+sQkc5u0dAV3Z7n5Avy1R/UjyBpRoYb7VhfX1mZTKsVfLPoOxMuvChyOAL4MZGR83N9emHoLFL1r/JsRlqtun/F8i7vsu3OmqpO9ZcexEh+jx1Hqt6qSizNto6bQkKQ5WOul0kfT9U4kIHcKe0EIsUmD2d6CYLXBIYuSitp3/Fq/yX/tUwm9tJED+J0g76g8ItFAznf+7nYe4MSh039ehOKXLW4v+w6624RYfmjCauNHb4HSmwE0n2y8XgLwat7Tj1hwV8Qc=
+
```
Comment (JacOng17) [inline .travis.yml]:
```suggestion

```

File: .travis.yml [hunk 3]
```diff
@@ -4,19 +4,49 @@ cache:
...
+    - google-chrome-stable
+
+env:
+  global:
+    - RAILS_ENV=test
+    - OPENFISCA_URL=https://api.rules.nz
+    - GITHUB_URL=https://github.com/ServiceInnovationLab/openfisca-aotearoa
+    - OPENFISCA_GIT_CLONE_URL=https://github.com/ServiceInnovationLab/openfisca-aotearoa
+    - secure: T7+npfKb6EW0G++OP4C91y2WMrMYyOrozeNT1WPCuedd5vKdILSskMz5LBav54F0J6OBbwozH82e0xclMTa23VuOrc9805tEcwtSRdDYb/mIb7F6gEyseqLIk/TZvDfmB5Yrrji0IVX9s5q0OKfVVb0rTrgh87pSRW2YnYAu41k6v2pK/OquixQX0sogblOLfKYprqh4h3Ay6+1IIDmC/lg0XSe+0xRFBZ5FJk0ycStu74xM+qbphIFgzd6q2kChFqRrOporeCgN3WSAFsQd7QneLITN3eucH9AR0OC9Udpuxr8tvAqTfy7Lo3ogj9PTYjD60ObQjcr212d/isSaZd6Zk2wHH35bzsZU3Y5f6vsILiZVJY2rDPkf7A6jMrWSPw8jd/CRizFjedveF9jCy3J2viwNI23y4Ce+sQkc5u0dAV3Z7n5Avy1R/UjyBpRoYb7VhfX1mZTKsVfLPoOxMuvChyOAL4MZGR83N9emHoLFL1r/JsRlqtun/F8i7vsu3OmqpO9ZcexEh+jx1Hqt6qSizNto6bQkKQ5WOul0kfT9U4kIHcKe0EIsUmD2d6CYLXBIYuSitp3/Fq/yX/tUwm9tJED+J0g76g8ItFAznf+7nYe4MSh039ehOKXLW4v+w6624RYfmjCauNHb4HSmwE0n2y8XgLwat7Tj1hwV8Qc=
 before_install:
```
Comment (JacOng17) [inline .travis.yml]:
```suggestion

before_install:
```

**Comment by JacOng17** [comment]
Removed Percy which causes PR to not be able to get merged.

File: .rubocop.yml [hunk 4]
```diff
@@ -2,9 +2,13 @@
...
 
+require:
+  - rubocop-rspec
+  - rubocop-rails
+
 # We do not typically use/need class documentation
-Documentation:
-  Enabled: false
+# Documentation:
+#   Enabled: false
```
Comment (Br3nda) [inline .rubocop.yml]: This cab be deleted I think

**Pull request merged**